tabpy-server/setup.py

```diff
@@ -46,8 +46,7 @@
         'requests',
         'singledispatch',
         'six',
-        'tornado==5.1.1',
-        'Tornado-JSON',
+        'tornado',
```

Comment:
We need to guarantee all the tests pass before upgrade tornado to the latest version.
Can't just install the latest one.

Reply:
What version do you want to pin it to?